From 872f049c695bef40a8e976a58ba6689f24a976c3 Mon Sep 17 00:00:00 2001
From: Hai Nguyen Pham <hai.pham.ud@renesas.com>
Date: Thu, 11 Oct 2018 18:31:05 +0700
Subject: [PATCH 698/909] Revert "arm64: dts: renesas: r8a7795: Connect
 Ethernet-AVB to IPMMU-DS0"

commit e416344eb0662648ce1d4d765d27ef8784684e4d from
git://git.kernel.org/pub/scm/linux/kernel/git/horms/renesas-bsp.git

This reverts commit 8cd8833fa32dddfbad56967fd858579282cbb131
to keep IPMMU address translation for Ethernet-AVB is disabled
by default, maintain the same IPMMU support environment with
previous released R-Car Linux BSP.

Signed-off-by: Hai Nguyen Pham <hai.pham.ud@renesas.com>
Signed-off-by: Meng Li <Meng.Li@windriver.com>
---
 arch/arm64/boot/dts/renesas/r8a7795.dtsi | 1 -
 1 file changed, 1 deletion(-)

diff --git a/arch/arm64/boot/dts/renesas/r8a7795.dtsi b/arch/arm64/boot/dts/renesas/r8a7795.dtsi
index 43f9295d3a51..cb9ce2db0e18 100644
--- a/arch/arm64/boot/dts/renesas/r8a7795.dtsi
+++ b/arch/arm64/boot/dts/renesas/r8a7795.dtsi
@@ -1660,7 +1660,6 @@
 			power-domains = <&sysc R8A7795_PD_ALWAYS_ON>;
 			resets = <&cpg 812>;
 			phy-mode = "rgmii";
-			iommus = <&ipmmu_ds0 16>;
 			#address-cells = <1>;
 			#size-cells = <0>;
 			status = "disabled";
-- 
2.17.1

